src/sb_pointcloud_processing/test/TestUtils.h

```diff
@@ -47,8 +47,14 @@ class TestUtils {
     static void addLineToPointCloud(LineArgs args,
```

Comment:
We should probably add a Javadoc style function comment here.

Reply:
done